<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FactuurKeuzeLijst</class>
 <widget class="QDialog" name="FactuurKeuzeLijst">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>826</width>
     <height>46</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="rbtnLaatste">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>104</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Laatste 100.</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtnNummer">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>10</y>
      <width>104</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Op nummer:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtnDatum">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>10</y>
      <width>104</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Op datum:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dtpDatum">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>106</width>
      <height>25</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtnNaam">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>104</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Op naam:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbNaam">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>tbNaam</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>5</y>
      <width>41</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/icons/icons/zoom.png</normaloff>:/icons/icons/zoom.png</iconset>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbNummer">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>10</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tvFactuurLijst">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>55</y>
     <width>826</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
